specs/trust-store-trust-policy.md

```diff
@@ -301,6 +344,51 @@ Signature verification levels provide defined behavior for each validation e.g.
     }
 ```
 
+#### Timestamp Countersignature Verification details
+
+Sufficient and necessary conditions to trigger timestamp countersignature verification:
+  - As the overall switch, `authenticTimestamp` under `signatureVerification` of trust policy is NOT marked as `skip`.
+  - The signing scheme is [`notary.x509`](./signing-scheme.md/#notaryx509).
+  - `tsa` trust store type is configured under `trustStores` field of the corresponding trust policy statement.
+  - `verifyTimestamp` under `signatureVerification` of trust policy is not set or set to `always`; OR it's set to `afterCertExpiry` and at least one certificate in the signing certificate chain has expired at the time of verification.
+
+See the flow chart below for a better understanding:
+```mermaid
```

Comment:
nit: flow chart is really big in size, can we restrict its size?

Reply:
I've actually tried several ways to change the size of the entire graph but with no luck in GitHub markdown. The graph does provide a shrink button though, one could shrink the graph to fit the screen while reading the spec.